src/en/clouds-manual.md

```diff
@@ -1,7 +1,7 @@
-Title: Adding a manual cloud
+Title: Using the Manual cloud with Juju
```

Comment:
I don't know why you changed this to 'the' manual cloud. It makes it sound like some sort of entity. We don't say 'the MAAS cloud'. The original phrasing wasn't great either. Maybe 'Creating a manual cloud to use with Juju' ?

Reply:
Another terminology fail. We are left with the generic term of "manual" to work with. We don't say "the MAAS cloud" because MAAS is a proper noun used as an adjective, but "manual" is not a proper noun. So "a manual cloud" is devoid of meaning when used on its own and can even be construed as a cloud one has built manually (say with OpenStack). Also, this type of cloud has been flagged as "special" so using the term in a generic way is contradictory. By saying "the Manual cloud" we get around all this. One could argue that "*a* Manual cloud" could suffice but some may see it as a typographical error.